### PR TITLE
Created Character class

### PR DESCRIPTION
**Pull request by rightonhana**
Added an abstract Character class with its attributes and methods

### CODE REVIEW

**Comment by ducampcarlos** [comment]
Looks good

**Comment by ducampcarlos** [comment]
Everything seems fine.
But there's two details that are bothering me.
First thing, it should be baseAttack, not baseAtack (line 9 and the getters and setters are writen that way).
And since we're going to have the attributes maxLevel, baseAttack, baseDefense, and baseHp hardcoded since the program initiates, do we really need the Setters for those attributes?
We're not going to need those methods. Or on the other hand, we could use them as private, and instead of having them hardcoded, we could set them in the constructor.

**Comment by AlejandroWoloch** [comment]
there are 2 thing that bother me, I think we shouldn't have a setter for count, because you shouldn't change it anytime.
The other thing is that we should add some validations, like you should not be able to set a level higher than the maxlevel.
About what carlos said, for me it's okey to have thoes setters of statics attribiutes just in case we decide to modified them to male stronger or weaker monsters.

**Comment by AlejandroWoloch** [comment]
oh! another thing that I noticed: in calculateAttack you used Character.baseHP instead of level

**Comment by rightonhana** [comment]
Hi guys! Sorry for the delay.
Remember that you can add a comment in the code, in the exact line that you found the error. Try to use it! It's more fast and easy to read what I need to fix or change in the code,

Today I will be apply the fixes, thanks for the feedback!

File: src/Character.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+	// Methods
+	/**
+	 * Abstract method to calculate the attack (damage) of a character
+	 * 
+	 * This method will be defined and implemented by the child classes
+	 * */
+	public short calculateAttack() {
+		short random = (short)(Math.random() * (1 - 0)); 
+		return (short) (Character.baseAtack  + ( Character.baseAtack * random ) * Character.baseHp);
```
Comment (AlejandroWoloch) [inline src/Character.java]: Here it should be this.level instead of Character.baseHp

Comment (rightonhana) [reply src/Character.java]: Done!

File: src/Character.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+				return false;
+		} else if (!name.equals(other.name))
+			return false;
+		return true;
+	}
+
+	// Override toString method.
+	@Override
+	public String toString() {
+		return "Character [getId()=" + getId() + ", getLevel()=" + getLevel() + ", getName()=" + getName() + "]";
```
Comment (AlejandroWoloch) [inline src/Character.java]: I would modify toString so it doesn't show "getId() ="+ getId(), but "Id = " + getId() instead

Comment (rightonhana) [reply src/Character.java]: Done!

**Comment by AlejandroWoloch** [comment]
It's okey for me now

**Pull request merged**